cpp/tests/groupby/hash/type_info.hpp

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
How is this file different from cpp/src/groupby/hash/type_info.hpp ? Hard to compare them side-by-side in a code review.

Reply:
It's not much different, one is used for verifying the other. I debated this for a while, but I thought it'd be in poor form to use a file to test itself, because then it'll always come back as passing. Like using a tape measure to measure the accuracy of itself.